src/utils/trimErrorMessage.ts

```diff
@@ -0,0 +1,14 @@
+function splitAt(str: string, index: number) {
+  return index < 1 ? [str, ''] : [str.substring(0, index), str.substring(index)]
+}
+
+const MAX_LENGTH = 100
```

Comment:
We have a 32KB limit on individual events here so I don't mind going higher to 1024 chars or something that can more likely not trim on most messages

Reply:
Fixed by dc95df6c2f42d94c42d1020f7081a8f266ed113a